server2019/root/virtualization/v2/ConcreteJob_JobType.go

```diff
@@ -37,9 +38,9 @@ const (
 	// Shut_Down_Virtual_Machine enum
 	ConcreteJob_JobType_Shut_Down_Virtual_Machine ConcreteJob_JobType = 16
 	// Pause_Virtual_Machine enum
-	ConcreteJob_JobType_Pause_Virtual_Machine ConcreteJob_JobType = 26
+	ConcreteJob_JobType_Pause_Virtual_Machine ConcreteJob_JobType = 20
```

Comment:
You are not supposed to change this file. It would be over written if re-generated

Reply:
If these enums are incorrect, create a local enum in virtualmachine.go and use it.

And Add comments as to why this enum is incorrect